axelrod/tests/unit/test_cooperator.py

```diff
@@ -43,13 +39,11 @@ class TestTrickyCooperator(TestPlayer):
     }
 
     def test_strategy(self):
-        """Starts by cooperating."""
+        # Starts by cooperating.
```

Comment:
I'm indifferent, although in terms of PEP8 standards is a docstring not what is usual here?